api/envoy/extensions/stat_sinks/open_telemetry/v3/open_telemetry.proto

```diff
@@ -55,4 +70,13 @@ message SinkConfig {
   // "pre", the full stat name will be "pre.foo.bar". If this field is not set, there is no
   // prefix added. According to the example, the full stat name will remain "foo.bar".
   string prefix = 6;
+
+  // The custom conversion from a stat to a metric. Currently, the only supported input is
+  // ``envoy.extensions.matching.common_inputs.stats.v3.StatFullNameMatchInput`` and the only support action is
+  // ``envoy.extensions.stat_sinks.open_telemetry.v3.SinkConfig.ConversionAction``.
+  //
+  // If set, the sink will aggregate metrics including the ones not defined in this field as follows:
```

Comment:
I thought we agreed that we're changing the behavior to aggregate, regardless of whether this field is set?

Reply:
I still have not heard a single example of what this would actually break.  As far as I can tell, the change here is semantically equivalent, and there's no reason not to do it.

If you're concerned that this could be a problem, then putting a runtime flag in Envoy would be fine.  But I don't think we should expose this detail in the xDS API.

Reply:
My main concern here is that I don't want to enshrine Envoy's current non-aggregating behavior as the defined behavior of the xDS API.  By documenting the fact that aggregation will happen when the matcher field is present, we are implicitly saying that it won't happen when the matcher field is not present, and that's my objection.  I consider it a bug that Envoy was not already doing this aggregation, and I think fixing it shouldn't break anything that doesn't have its own bugs.

If you remove the documentation that setting this matcher field causes aggregation to happen, then I'm fine with this PR.  The runtime field we're talking about here will affect the behavior only when the matcher field is *not* set, so in principle that can be done in a separate PR.